From b2ae5a8604d59e8ef99ad096fb8d813c10be9ba5 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Wed, 1 Apr 2015 17:23:43 +0800
Subject: [PATCH] MLK-10539-2: ARM: clk_imx7d: add audio_post_div clock to
 init list

Add audio_post_div clock to init list

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
---
 arch/arm/mach-imx/clk-imx7d.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-imx/clk-imx7d.c b/arch/arm/mach-imx/clk-imx7d.c
index cc0d4a0..387e0d6 100644
--- a/arch/arm/mach-imx/clk-imx7d.c
+++ b/arch/arm/mach-imx/clk-imx7d.c
@@ -370,6 +370,7 @@ static int const clks_init_on[] __initconst = {
 	IMX7D_PLL_SYS_PFD7_CLK, IMX7D_DRAM_PHYM_ROOT_CLK, IMX7D_DRAM_ROOT_CLK,
 	IMX7D_DRAM_PHYM_ALT_ROOT_CLK, IMX7D_DRAM_ALT_ROOT_CLK,
 	IMX7D_AHB_CHANNEL_ROOT_CLK, IMX7D_NAND_USDHC_BUS_ROOT_CLK, IMX7D_PLL_VIDEO_POST_DIV,
+	IMX7D_PLL_AUDIO_POST_DIV,
 	};
 
 static struct clk_onecell_data clk_data;
-- 
1.8.0

